exo-go/src/types/secrets.go

```diff
@@ -0,0 +1,49 @@
+package types
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// Secrets map contains maps from secret keys to values
+type Secrets map[string]string
+
+// NewSecrets creats a Secrets map from a .tfvars string
+// a="b"
+// c="d" ->
+// {a:b, c:d}
```

Comment:
Well, since the golinter requires we have comments I'll leave it but change it to be more clear.